checklist.html

```diff
@@ -1,242 +1,1029 @@
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          This includes sections of related content, items displayed in a grid, or sibling <code>a</code> elements.
+        </p>
+      </div>
+    </details>
+  </div>
+</section>
+
+<section class="article-section resources-section checklist-section" id="section-controls" aria-labelledby="label-controls">
+  <h2 id="label-controls">Controls</h2>
+  <p>
+    Controls are interactive elements such as links and buttons that let a person navigate somewhere or take action.
+  </p>
+  <div class="checklist">
+    <details id="section-controls-01">
+      <summary>
+        Use the <code>a</code> element for links.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          Links are used to load a new page, or shift keyboard focus from one element to another.
+        </p>
+      </div>
+    </details>
+    <details id="section-controls-02">
+      <summary>
+        Use the <code>button</code> element for buttons.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          Buttons are used to submit data or perform an on-screen action which does not shift keyboard focus.
+        </p>
+      </div>
+    </details>
+    <details id="section-controls-03">
+      <summary>
+        Provide a skip link and make sure that it is visible when focused.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html">2.4.1 Bypass Blocks</a>
+        </p>
+        <p class="checklist-description">
+          A <a href="/posts/skip-nav-links/">skip link</a> can be used to provide quick access to the main content of a page or view. This allows a person to easily bypass globally repeated content such as a website's primary navigation, or persistent search widget.
+        </p>
+      </div>
+    </details>
+    <details id="section-controls-04">
+      <summary>
+        Identify links that open in a new window, or links that navigate to external sites.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html">3.2.2 On Input</a>
```

Comment:
Not doing this would not actually fail this SC.  more of an advisory technique / best practice, which is actually indicated in the understanding link, which then links to: https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G201

Reply:
Is there another SC it would fail?

Reply:
i don't think so.  More of a best practice.  Especially regarding external links.

Reply:
So I think our options here are to:

1. Leave it be and have the SC be a little bit of a white lie, because it is an important thing to think about.
1. Remove it, as it's not technically a failure, which is not in keeping with our other checklist items.
1. Add a note to the end about what you mentioned, which I think kind of is like the second option.

I'm personally leaning towards removal. I want this checklist to be airtight.

Reply:
I think this could stay in if you remove mentioning external links. Opening in a new window you could link over to http://www.w3.org/TR/WCAG20-TECHS/G201.html as a reasoning/ technique

Reply:
I think this could stay in if you remove mentioning external links. Opening in a new window you could link over to http://www.w3.org/TR/WCAG20-TECHS/G201.html as a reasoning/ technique

Reply:
Done.